templates/base.html

```diff
@@ -5,9 +5,11 @@
     <link rel="icon" type="image/png" href="{{ STATIC_URL }}img/favicon.ico" />
     <!-- Set the viewport width to device width for mobile -->
     <meta name="viewport" content="width=device-width" />
-    <title>
-      {% block title %}Home{% endblock %}
-      - The Political Memory of {{ request.site.SITE_NAME }}</title>
+      <title>
+      {% block title %}
+          Home - The Political Memory of {{ request.site.SITE_NAME }}
+      {% endblock %}
```

Comment:
Should we remove extra spaces in the title tag ?

Ie. `<title>{% block ... %}{% endblock %}</title>`

https://www.w3.org/Provider/Style/TITLE.html